docs/data/material/components/text-fields/text-fields.md

```diff
@@ -224,20 +224,17 @@ Make sure that the input is larger than the label to display correctly.
 
 ### type="number"
 
-Inputs of type="number" have potential usability issues:
+:::warning
+We do not recommend using type="number" with a Text Field due to potential usability issues:
```

Comment:
Last tiny nitpick otherwise I think this is ready to go!

```suggestion
We do not recommend using `type="number"` with a Text Field due to potential usability issues:
```